pkg/api/persistentvolumeclaim/util_test.go

```diff
@@ -74,52 +74,38 @@ func TestDropDisabledSnapshotDataSource(t *testing.T) {
 	// Ensure that any data sources aren't enabled for this test
 	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.AnyVolumeDataSource, false)()
 
-	for _, enabled := range []bool{true, false} {
-		for _, oldpvcInfo := range pvcInfo {
-			for _, newpvcInfo := range pvcInfo {
-				oldPvcHasDataSource, oldpvc := oldpvcInfo.hasDataSource, oldpvcInfo.pvc()
-				newPvcHasDataSource, newpvc := newpvcInfo.hasDataSource, newpvcInfo.pvc()
-				if newpvc == nil {
-					continue
-				}
+	for _, oldpvcInfo := range pvcInfo {
+		for _, newpvcInfo := range pvcInfo {
+			oldpvc := oldpvcInfo.pvc()
+			newPvcHasDataSource, newpvc := newpvcInfo.hasDataSource, newpvcInfo.pvc()
```

Comment:
do you actually need this bool or can you infer from the pvc object?

Reply:
I'll remove this.